File: core/src/de/fatox/meta/api/Logger.java
package de.fatox.meta.api;

public interface Logger {
    enum LogLevel {
        TRACE, DEBUG, INFO, WARNING, ERROR
    }

    void trace(String tag, String text);

    void info(String tag, String text);

    void debug(String tag, String text);

    void warning(String tag, String text);

    void error(String tag, String text);
}

File: runtime/src/de/fatox/meta/api/Logger.kt
package de.fatox.meta.api

interface Logger {
    enum class LogLevel {
        TRACE, DEBUG, INFO, WARNING, ERROR
    }

    fun trace(tag: String, text: String)

    fun info(tag: String, text: String)

    fun debug(tag: String, text: String)

    fun warning(tag: String, text: String)

    fun error(tag: String, text: String)
}
